code/modules/projectiles/guns/launcher/bows/bow_string.dm

```diff
@@ -0,0 +1,33 @@
+/obj/item/bowstring
+	name = "bowstring"
+	// TEMP, TODO PROPER ICON
+	icon = 'icons/obj/fishing_line.dmi'
+	icon_state = ICON_STATE_WORLD
+	// END TEMP
```

Comment:
nitpicking here but `icon_state` will presumably still be `ICON_STATE_WORLD` even once it gets a real icon, so the temp should probably just be on the icon var :p